hs/Response/Post.hs

```diff
@@ -115,18 +115,14 @@ checkPost =
                     H.input ! A.type_ "text"
                     H.input ! A.type_ "text"
             H.div ! A.id "spec_extra" $ do
-                H.p ! A.class_ "code" $ "Any of the following: 300+ level CSC course; MAT: 235/237/257, any 300+\
-                                        \ except for 329, 390, & 391; STA: 248, 261, any 300+; ECE: 385H/489H;\
-                                        \ BCB: 410H/420H/430Y (1.5 FCEs)"
+                H.p ! A.class_ "code" $ "Any of the following: 300+ level CSC course; MAT: 235/237/257, any 300+  except for 329, 390, & 391; STA: 248, 261, any 300+; ECE: 385H/489H;  BCB: 410H/420H/430Y (1.5 FCEs)"
                 H.div ! A.id "specextra" ! A.class_ "more-info" $ do
                     H.input ! A.type_ "text"
                     H.input ! A.type_ "text"
                     H.input ! A.type_ "text"
                     H.input ! A.type_ "text"
             H.div ! A.id "spec_misc" $ do
-                H.p ! A.class_ "code" $ "Any from this list: CSC301H, CSC318H, CSC404H, CSC411H, CSC418H, CSC420H, \
-                    \CSC428H, CSC454H, CSC485H, CSC490H, CSC491H, CSC494H, or PEY (0.5 FCEs) \
-                    \ ** Note: Type 'PEY' for Check my POSt to recognize it **"
+                H.p ! A.class_ "code" $ "Any from this list: CSC301H, CSC318H, CSC404H, CSC411H, CSC418H, CSC420H,  CSC428H, CSC454H, CSC485H, CSC490H, CSC491H, CSC494H, or PEY (0.5 FCEs)   ** Note: Type 'PEY' for Check my POSt to recognize it **"
```

Comment:
Same comment as HaXml - please don't include these changes in this pull request.